.github/workflows/release.yml

```diff
@@ -46,3 +46,18 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
+
+  trivy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Run Trivy vulnerability scanner
```

Comment:
```suggestion
        uses: actions/checkout@v2
        
      - name: Set Tag
        run: |
          echo "::set-env name=TAG::${GITHUB_REF#refs/*/v}"
          echo "::set-env name=RELEASE_TAG::${TAG}"
        
      - name: Run Trivy vulnerability scanner
```

Reply:
This also need to be taken up, else `env.RELEASE_TAG`  wont be set.

Reply:
But we have already set tags on lines 28:31

Reply:
The envs are specific to the job. So those are available only for the `linux-utils` job. 
We need the envs for the `trivy` job

Reply:
Ok 👍.